src/ai/default/aspect_attacks.cpp

```diff
@@ -141,7 +141,7 @@ void aspect_attacks_base::do_attack_analysis(const map_location& loc,
 
 	ai::manager::get_singleton().raise_user_interact();
 	const int max_attack_depth = 5;
-	if(cur_analysis.movements.size() >= std::size_t(max_attack_depth)) {
+	if(cur_analysis.movements.size() >= static_cast<std::size_t>(max_attack_depth)) {
```

Comment:
Fix the previous line instead, these two lines are the only place this constant is used.

Reply:
Updated.